src/query.ts

```diff
@@ -500,7 +500,7 @@ export class Query<
    */
   is<Column extends GetMethodColumn<DB, TableName>>(
     column: Column,
-    value: HorizontalFilterValue<DB, TableName, Column>,
+    value: null | true | false,
```

Comment:
We should later add support for `unkown` which is from what I understand a special PostgREST boolean value
https://postgrest.org/en/v12/references/api/tables_views.html#horizontal-filtering